src/lib.rs

```diff
@@ -191,6 +191,22 @@ impl<V> VecMap<V> {
         }
     }
 
+    /// Trims the `VecMap` of any excess capacity.
```

Comment:
don't you think "excess capacity" could also describe `None`s that may remain at the end of the array?